packages/Core/theme/tabs.ts

```diff
@@ -75,5 +81,22 @@ export const getTabs = (theme: WuiTheme): ThemeTabs => {
         width: borderWidths.lg,
       },
     },
+    size: {
+      sm: {
+        paddingBottom: toRem(9),
+        marginRight: toRem(12),
+      },
+      md: {
+        paddingBottom: toRem(15),
+        marginRight: toRem(24),
+      },
```

Comment:
Tou need to get this values from theme

Reply:
when value doesn't exist in theme (ie. 9, 15), I go for the closest one or I keep toRem() with requested value ?

Reply:
Closest, i think the component is not fully good on design team